doc/design/service-discovery.md

```diff
@@ -262,53 +262,45 @@ myListener.attachEvent(caproSubscriber, DATA_RECEIVED, createNotificationCallbac
 #### Event-based notification: Alternative D
 
 ```cpp
-class DiscoveryInfo
+class ServiceDiscovery
```

Comment:
Because of those kind of changes I would argue strongly against code (or at least much code, apart from algorithmic snippets and struct definitions) in those design documents. Can stay like this but should be considered in the future.

Reply:
I disagree, design documents always go along with code and tests. In my experience things can never be developed alone.